generator/internal/mixin_utils_test.cc

```diff
@@ -206,63 +239,58 @@ TEST_F(MixinUtilsTest, ExtractMixinProtoPathsFromYaml) {
 
 TEST_F(MixinUtilsTest, GetMixinMethods) {
   FileDescriptor const* file = FindFile("test/v1/service1.proto");
-  EXPECT_THAT(file, NotNull());
+  ASSERT_THAT(file, NotNull());
 
   ServiceDescriptor const* service = file->service(0);
-  EXPECT_THAT(service, NotNull());
+  ASSERT_THAT(service, NotNull());
 
   auto const& mixin_methods = GetMixinMethods(service_config_, *service);
   EXPECT_EQ(mixin_methods.size(), 3);
```

Comment:
nit: this is unnecessary now that we have the `ElementsAre()` with three matchers.

Reply:
Removed.